src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorBuildStep.groovy

```diff
@@ -43,7 +45,11 @@ class IqPolicyEvaluatorBuildStep
 {
   String iqStage
 
-  String iqApplication
+  String applicationSelectTypePost
```

Comment:
Wondering if this can be encapsulated in another model - @ja98200wilcox mentioned he had difficulty doing so, looking into it.